src/links/links.macros.html

```diff
@@ -19,7 +19,7 @@
 <a class="link{% if icon %} link--icon{% endif %}{% if class %} {{ class }}{% endif %}"{% if id %} id="{{ id }}"{% endif %} href="{{ href }}">
   {{ content }}
   {% if icon %}
-    {{ icon_macro(icon) }}
+    {{ icon_macro(icon, size="tiny") }}
   {% endif %}
 </a>
 {% endmacro %}
```

Comment:
Not a super big deal in this context, but worth noting so as well work on improving our commits that the update to macro doesn't fit the description of this commit. Perhaps could have been rebased/squashed into the commit the added the macro file. 

Overall though, :100: on small, single purpose commits. <3'ing it